ccd-definition/ComplexTypes.json

```diff
@@ -1556,5 +1556,206 @@
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "RespondentParty",
+    "ListElementCode": "respondent_border_1",
+    "FieldType": "Label",
+    "ElementLabel": "---",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "RespondentParty",
+    "ListElementCode": "relationshipLabel",
+    "FieldType": "Label",
+    "ElementLabel": "## Relationship to the child",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "RespondentParty",
+    "ListElementCode": "relationshipToChild",
+    "FieldType": "TextArea",
+    "ElementLabel": "What is the respondent’s relationship to the child or children in this case?",
+    "HintText": "Include: the name of the child or children, the respondent’s relationship to them and any details if you’re not sure this person has parental responsibility",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "RespondentParty",
+    "ListElementCode": "respondent_border_2",
+    "FieldType": "Label",
+    "ElementLabel": "---",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "RespondentParty",
+    "ListElementCode": "contactDetailsHidden",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Do you need contact details hidden from anyone?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "RespondentParty",
+    "ListElementCode": "contactDetailsHiddenReason",
+    "FieldType": "TextArea",
+    "ElementLabel": "Give reason",
+    "FieldShowCondition": "contactDetailsHidden=\"Yes\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "RespondentParty",
+    "ListElementCode": "proceedingsLabel",
+    "FieldType": "Label",
+    "ElementLabel": "## Ability to take part in proceedings",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "RespondentParty",
+    "ListElementCode": "litigationIssues",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "LitigationCapacityIssues",
+    "ElementLabel": "Do you believe this person will have problems with litigation capacity (understanding what's happening in the case)?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "RespondentParty",
+    "ListElementCode": "litigationIssuesDetails",
+    "FieldType": "TextArea",
+    "ElementLabel": "Give details, including assessment outcomes and referrals to health services",
+    "FieldShowCondition": "litigationIssues=\"YES\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "RespondentParty",
+    "ListElementCode": "respondent_border_3",
+    "FieldType": "Label",
+    "ElementLabel": "---",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "RespondentNew",
+    "ListElementCode": "leadRespondentIndicator",
```

Comment:
We should make it mandatory if it is mandatory in schema design.

Same for other mandatory fields.

Reply:
No - new direction is to make this field hidden and populate in an AboutToSubmit callback. Our users have the following understanding of respondents:

The first respondent (mother) is [name]
The second respondent (father/father of ……………………………) is [name]
The third respondent(s) is/are (the children) by their children’s guardian [name]

Taken from a template provided by Sinead